flowgate-common/src/main/java/com/vmware/flowgate/common/model/PduInlet.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+*/
+package com.vmware.flowgate.common.model;
+
+public class PduInlet {
+
+   private Long id;
+   private Long pduId;
+   private Integer ordinal;
+   private String formatedName;
+   private boolean pduPower;
```

Comment:
what does pduPower mean?  


also you should pay attention to the pue when calculate the real power usage.
 - (boolean) pue_it readonly
Whether readings for this inlet should be included in PUE IT energy.
- (boolean) pue_total readonly

Whether readings for this inlet should be included in PUE total energy.

Reply:
Done.